Documentation/ApiOverview/Bootstrapping/Index.rst

```diff
@@ -87,7 +87,7 @@ contexts:
 
 * ``Production`` (default) - should be used for a live site
 * ``Development`` - used for development
-* ``Testing`` - is used for functional tests
+* ``Testing`` - is only used internally when executing TYPO3 tests. It must not be used otherwise.
```

Comment:
Please clearify:
* ``Testing`` - is only used internally when executing TYPO3 **Core** tests. It must not be used otherwise.